tests/test_profile.py

```diff
@@ -208,7 +208,7 @@ def test_private_groups_field_as_public_when_logged_in(self, base_url, selenium,
     @pytest.mark.credentials
     @pytest.mark.nondestructive
     def test_private_groups_field_when_not_logged_in(self, base_url, selenium, private_user):
-        page = Profile(selenium, base_url, username=private_user['name']).open()
+        page = Profile(selenium, base_url, username=private_user['username']).open()
```

Comment:
KeyError: 'username'. Can you please update the key to "name"?

Reply:
In my testing this needed to be the username. Using the name was returning a page not found. It's possible that username and name were identical in the previous variables file?

Reply:
Oh, I understand why it's this happening. Name and username were indeed identical in the variables file.
Name doesn't work anymore because it is set to be visible to "Mozillians" only (instead of "Public"), so the field is not shown for users that are not logged in and search returns no results. Username field is "Public" by default and it cannot be changed from "Public". So I guess it's good for this test to use Username.